lib/iaas/BackupStore.js

```diff
@@ -181,6 +182,10 @@ class BackupStore {
           const filename = data.name;
           return this
             .deleteServiceBackup(data, options)
+            .catch(BaseCloudClient.providerErrorTypes.NotFound, err => {
+              logger.warn(`Snapshot/ backup data not found while deleting backup guid ${data.backup_guid}.
```

Comment:
if you have inline evaluation , can you please change the below to `...Error:${err.message}` or if your intention was to print stack trace then the method arg to log should be error object and not err.message.